### ISSUE TITLE
Add support for bigip in "virtualAddresses"

### ISSUE THREAD

**Issue by rsawant23** [F5Networks/f5-appsvcs-extension#174]
##### Do you already have an issue opened with F5 support?
I will open a case with f5 shortly after this and reference this issue in it

##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
3.11.0-3
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.0.2
  Build       0.0.6
  Edition     Point Release 2
  Date        Tue Jan 16 08:46:28 PST 2018
```

##### SUMMARY

```
When I try to reference bigip in my virtualAddress block of my PATCH declaration like so 
                "virtualAddresses": [{"bigip": "/Common/<ip-address>"}],
I get the following error:
{
    "code": 500,
    "declarationFullId": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d|QA1",
    "message": "Cannot read property 'split' of undefined"
}
```

##### STEPS TO REPRODUCE

```
[
    {
        "path": "/STG_Fix/lba",
        "op": "add",
        "value": {
            "class": "Application",
            "label": "<label>",
            "template": "generic",
            "<vip-name>": {
                "class": "Service_TCP",
                "snat": {
                    "bigip": "/Common/Floating_SNAT"
                },
                "iRules": [
                    {
                        "bigip": "/Common/<iRule-name>"
                    }
                ],
                "virtualAddresses": [{"bigip": "/Common/<ip-address>"}],
                "virtualPort": 39191,
                "pool": "/STG_Fix/Shared/<pool-name>"
            }
        }
    },    
    {
        "path": "/STG_Fix/Shared/<pool-name>",
        "op": "add",
        "value": {
            "class": "Pool",
            "loadBalancingMode": "least-connections-member",
            "minimumMonitors": 2,
            "monitors": [
                {
                    "bigip": "/Common/tcp_120"
                },
				{
                    "bigip": "/Common/KILL-SWITCH-MONITOR"
                }
            ],
            "members": [
                {
                    "servicePort": <port>,
                    "addressDiscovery": "fqdn",
                    "hostname": "<server-name>.dmz.root"
                }
            ]
        }
    }
]
```

##### EXPECTED RESULTS

```
The vip inside a partition references a virtal address in common
```

##### ACTUAL RESULTS

```
I get the following error

{
    "code": 500,
    "declarationFullId": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d|QA1",
    "message": "Cannot read property 'split' of undefined"
}
```

**Comment by dstokesf5** [comment]
Was the virtual address originally declared in AS3, or is the virtual address being managed outside of AS3?

**Comment by rsawant23** [comment]
Hi Daniel,

The virtual address is managed outside of AS3.

Thanks,
Rugved.

**Comment by dstokesf5** [comment]
Thank you for your input. I have added AUTOTOOL-859 to our internal product backlog to track this feature request.

**Comment by dstokesf5** [comment]
This was resolved in the 3.19.0 release.
